core/src/main/java/org/apache/iceberg/puffin/StandardBlobTypes.java

```diff
@@ -26,4 +26,17 @@ private StandardBlobTypes() {}
    * href="https://datasketches.apache.org/">Apache DataSketches</a> library
    */
   public static final String APACHE_DATASKETCHES_THETA_V1 = "apache-datasketches-theta-v1";
+
+  /**
+   * A serialized form of HIVE_COLUMN_STATS_OBJ. The full list of available stats are provided in the
+   * <a href="https://cwiki.apache.org/confluence/display/Hive/StatsDev#StatsDev-ColumnStatistics">
+   *     Hive columns stats wiki </a>
+   */
+  public static final String HIVE_COLUMN_STATS_OBJ = "column-statistics-obj";
```

Comment:
should this be named `hive-column-statistics-obj`?

Reply:
Yes.
I have updated the name to hive-column-statistics-obj.